docs/antrea-network-policy.md

```diff
@@ -547,12 +577,14 @@ particular Namespaces.
 
 **namespaces**: A `namespaces` field allows users to perform advanced matching on
 Namespace objects which cannot be done via label selectors. Currently, the
-`namespaces` field has only one matching strategy, `Self`. If set to `Self`, it indicates
-that the corresponding `podSelector` (or all Pods if `podSelector` is not set)
+`namespaces` field has two matching strategies, `Self` and `NotSelf`. If set to `Self`,
+it indicates that the corresponding `podSelector` (or all Pods if `podSelector` is not set)
 should only select Pods belonging to the same Namespace as the workload targeted
```

Comment:
"the same Namespace of the the target Pods".

I feel we can remove "(either through a policy-level AppliedTo or a rule-level Applied-To) by the current
ingress or egress rule", which might be more confusing.